contributed_definitions/nyaml/NXfit.yaml

```diff
@@ -0,0 +1,263 @@
+category: base
+doc: |
+  Description of a fit procedure.
+symbols:
+  doc: |
+    The symbols used in the schema to specify e.g. dimensions of arrays.
+  n_points: |
+    Number of discretization points inside the fitting region.
+  n_independent: |
+    Number of independent axes for this fit procedure.
```

Comment:
The n_points could be different for each independent axis, no?

Reply:
agreed, as I said above. This needs to be cleaned.